### PR TITLE
Docs updates

### PR DESCRIPTION
**Pull request by mnonnenmacher**
Please see the commit messages for details.

---
This change is [Reviewable](https://reviewable.io/reviews/heremaps/oss-review-toolkit/1316)

### CODE REVIEW

File: docs/GettingStarted.md [hunk 1]
```diff
@@ -259,18 +259,38 @@ The `scanner` writes a new ORT result file to `[scanner-output-path]/scan-result
 addition to the analyzer result from the input. This way belonging results are stored in the same place for traceability.
 If the input file already contained scan results they are replaced by the new scan results in the output.
 
-## 6. Generate a report
+## 6. Running the evaluator
```
Comment (sschuberth) [inline docs/GettingStarted.md]: Could you please add a reference to https://github.com/heremaps/oss-review-toolkit/issues/1236 to the commit message (but none that would auto-close that issue yet, as still more pieces are missing)?

File: docs/GettingStarted.md [hunk 2]
```diff
@@ -124,23 +124,32 @@ repository:
...
   config:
     ignore_tool_versions: false
     allow_dynamic_versions: true
-    remove_excludes_from_result: false
   # The result of the dependency analysis.
   result:
     # Metadata about all found projects, in this case only the mime-types package defined by the package.json file.
     projects:
-    - id: "NPM::mime-types:2.1.19"
+    - id: "NPM::mime-types:2.1.18"
```
[the remark this answers is not in the document]
Comment (mnonnenmacher) [reply docs/GettingStarted.md]: Approved, will rebase later.

[the remark this answers is not in the document]
Comment (sschuberth) [reply docs/GettingStarted.md]: Thanks, and merged.

[the remark this answers is not in the document]
Comment (mnonnenmacher) [reply docs/GettingStarted.md]: Done.

File: docs/GettingStarted.md (hunk 1 above)
Comment (mnonnenmacher) [reply docs/GettingStarted.md]: Done.

**Pull request merged**